### PR TITLE
Fixes for Bloch notebook

### PR DESCRIPTION
**Pull request by sezelt**
Import fixes and add a new `PointList` data convenience function

### CODE REVIEW

File: py4DSTEM/io/__init__.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from . import datastructure
+from .datastructure import DataCube, PointListArray, PointList
```
Comment (cophus) [inline py4DSTEM/io/__init__.py:2]: I changed it to:

from .datastructure import *

Is there any risk in importing all of them?

File: py4DSTEM/process/diffraction/crystal.py [hunk 2]
```diff
@@ -345,9 +361,7 @@ def from_unitcell_parameters(
 
         return Crystal.from_pymatgen_structure(structure)
 
-    def setup_diffraction(
-        self, accelerating_voltage: float
-    ):
+    def setup_diffraction(self, accelerating_voltage: float):
```
Comment (cophus) [inline py4DSTEM/process/diffraction/crystal.py:364]: I still think we should be able to skip specifying voltage here in many quick workflows - perhaps instead specify a default option for crystal on instantiation?

**Review by cophus — COMMENTED** [review]
Looks good - just check the specific data structures vs *

File: py4DSTEM/io/__init__.py (hunk 1 above)
Comment (sezelt) [reply py4DSTEM/io/__init__.py:2]: I think it is fine

File: py4DSTEM/process/diffraction/crystal.py (hunk 2 above)
Comment (sezelt) [reply py4DSTEM/process/diffraction/crystal.py:364]: OK I made it default to 300 kV if none is set, and give a warning in `generate_diffraction_pattern`

File: py4DSTEM/io/__init__.py (hunk 1 above)
Comment (sezelt) [reply py4DSTEM/io/__init__.py]: Made the change and pushed it

**Review by cophus — APPROVED** [review]
Looks good!

**Review by bsavitzky — APPROVED** [review]

**Pull request merged**